lib/src/stores/send/send_store.dart

```diff
@@ -154,6 +158,16 @@ abstract class SendStoreBase with Store {
     }
   }
 
+  Future<bool> isOpenaliasRecord(String name) async {
```

Comment:
We can check for not is the text is openalias record, firstly we can try to check is it domain url or not and then if it's domain url we can think that it's openalias. It can be more faster if you are checking is it openalias on change of each character in address text field.